starknet_devnet/util.py

```diff
@@ -243,3 +244,26 @@ def get_fee_estimation_info(tx_fee: int, gas_price: int):
 def warn(msg: str, file=sys.stderr):
     """Log a warning"""
     print(f"\033[93m{msg}\033[0m", file=file)
+
+
+class SuppressLogger:
+    """ctx mng to suppress logger"""
+
+    def __init__(self, logger):
+        self.logger = logger
+
+    def __enter__(self):
+        self.logger.disabled = True
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.logger.disabled = False
+
+
+client_spec = importlib.util.find_spec("services.external_api.client")
```

Comment:
Why is this imported like this?
What about
```python
import services.external_api.client as client_spec
```

And what even is this `services` module? I see it's not native to Python, but it's also not a dependency listed in `pyproject.toml`.

Reply:
Yes, my bad. Rewrote to check the existence of the target logger, not the module. 
The `services` module is associated with Everest, from which FeederGatewayClient is inherited.